src/doc/trpl/unsafe.md

```diff
@@ -127,6 +111,101 @@ The latter assumption allows the compiler to optimize more
 effectively. As can be seen, actually *creating* a raw pointer is not
 unsafe, and neither is converting to an integer.
 
+## Reference and pointer aliasing
+
+Several examples can give a better comprehension of the reference and pointer aliasing rules.
+
+First of all, using `unsafe` code to incorrectly circumvent and violate the
+restrictions on references leads to undefined behaviour. For example,
```

Comment:
Is there any way to “correctly” circumvent the aliasing rules? If not, I’d probably remove the word. I’d also remove either “circumvent” or “violate”, since they’re synonymous.

> First of all, using `unsafe` code to circumvent the restrictions on references leads to undefined behaviour.